CHANGELOG.md

```diff
@@ -6,26 +6,62 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 
+## [v17.0.0] 2023-08-09
+
+### MIGRATION notes
+
+- This release updates the `hashicorp/aws` provider required by Cumulus to `~> 5.0`
+  which in turn requires updates to all modules deployed with Core in the same stack
+  to use a compatible provider version.
+- This update is *not* compatible with prior stack states - Terraform will not
+  allow redeployment of a prior version of Cumulus using an older version of
+  the provider.  Please be sure to validate the install changeset is what you
+  expect prior to upgrading to this version.
+- Migration forward to v17 from v16+ is strongly
+  recommended, as the changeset from the v16.x.y baselines is relatively minimal
+  and has been tested.
+- Upgrading Cumulus to v17 from prior versions should only require the usual
+  terraform init/apply steps.  As always **be sure** to inspect the `terraform plan` or
+  `terraform apply` changeset to ensure the changes between providers are what
+  you're expecting for all modules you've chosen to deploy with Cumulus
+
+### Notable Changes
```

Comment:
Should this be a Breaking change? It could go either way I guess but it might be more of a warning if we call it breaking?

Reply:
The problem, really, is that Orca has a dependency on Core, not Core being dependent on Orca, *except* that we've coupled it into the dashboard calls via api/lib/orca.   There's a reasonable discussion over if this should be the case *at all* and not have Cumulus provide a clearing house API that queries other, foreign APIs, but that's a separate conversation/ architecture discussion.  

I suppose we should put the AWS provider update and the Orca changes in as a breaking change:  9e181528c8